src/gui/DatabaseSettingsWidget.cpp

```diff
@@ -149,3 +202,13 @@ void DatabaseSettingsWidget::truncateHistories()
         entry->truncateHistory();
     }
 }
+
+void DatabaseSettingsWidget::changeKdf(int index) {
```

Comment:
When changing the KDF, we need a different transform round default value. The AES default is way to much for Argon2. When you change the KDF from AES to Argon2, but forget to hit the Benchmark button, KeePassXC freezes for a minute or longer while calculating the new key.